bundle/Resources/translations/ezrepoforms_role.en.xlf

```diff
@@ -22,6 +26,30 @@
                 <source>role.create</source>
                 <target>Create a role</target>
             </trans-unit>
+            <trans-unit id="f69be0366fdb75154d6781d3d1d63590">
+                <source>role.policy.type</source>
+                <target>Policy type</target>
+            </trans-unit>
+            <trans-unit id="be06833a23ccaafde1d58b4876abe401">
+                <source>role.policy.all_modules</source>
+                <target>All modules</target>
+            </trans-unit>
+            <trans-unit id="51cb855a33cf4446b6ffa51308c3ed1f">
+                <source>role.policy.all_functions</source>
+                <target>All functions</target>
```

Comment:
Nitpick: Consider to make the "All *"  entries stand out a bit more in the dropdown? Maybe with brackets around them like `[All modules]`, or upper case, or indent other entries? Just a thought.